source/scale/opensearch-setup.rst

```diff
@@ -0,0 +1,30 @@
+AWS OpenSearch server setup
+============================
+
+.. include:: ../_static/badges/ent-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+AWS OpenSearch Service allows you to search large volumes of data quickly, in near real-time, by creating and managing an index of post data. The indexing process can be managed from the System Console after setting up and connecting an OpenSearch server. The post index is stored on the OpenSearch server and updated constantly after new posts are made. In order to index existing posts, a bulk index of the entire post database must be generated.
+
+Deploying AWS OpenSearch includes the following two steps: `setting up AWS OpenSearch <#set-up-aws-opensearch>`__, and `configuring Mattermost <#configure-mattermost>`_. 
+
+Set up AWS OpenSearch
+----------------------
+
+From Mattermost v9.11, beta support is available for `AWS OpenSearch v1.x and v2.x <https://opensearch.org/>`_. We highly recommend that you set up an AWS OpenSearch server on a separate machine from the Mattermost server.
+
+1. Download and install the latest release of AWS OpenSearch. See the `OpenSearch <https://docs.opensearch.org/docs/latest/install-and-configure/>`_ documentation for installation details.
+
+<additional AWS OpenSearch setup instructions here>
```

Comment:
@cwarnermm here should have some or all of the instructions provided in the doc but instead still has the placeholder.

Reply:
Oooops! Sorry @stafot! That commit didn't go through for some reason. My latest commit is now added to this PR and ready for your review.